packages/vanilla/src/index.ts

```diff
@@ -65,7 +65,8 @@ export default class SelectionArea extends EventTarget<SelectionEvents> {
                 startThreshold: {x: 10, y: 10},
                 scrolling: {
                     speedDivider: 10,
-                    manualSpeed: 750
+                    manualSpeed: 750,
+                    startScrollMargins: { x: 10, y: 10 }
```

Comment:
Shouldn't the default value be `0`? Otherwise there will always be a margin which may be unexpected. I'm more a friend of explicit configuration rather than providing a "plug-and-play" tool ^^